cmd/adapters/ingresses/ingresses.go

```diff
@@ -107,6 +116,32 @@ func (c *VampCloudAnansiIngressClient) PostIngress(ingress models.Ingress) (int6
 
 }
 
+func checkPreviewRoute(routes []*dto.Route) error {
+
+	re := regexp.MustCompile(`%%(.*)%%`)
+	for _, v := range routes {
+		path := strings.Split(v.Path, "/")
+
+		for _, v2 := range path {
+			result := re.FindSubmatch([]byte(v2))
+			if len(result) > 0 && !validPlaceholder(result[1]) {
```

Comment:
You can have more than a single occurrence of a placeholder

Reply:
how?